src/App.tsx

```diff
@@ -81,15 +81,15 @@ const extendConsole = () => {
         window.console = {};
       }
       if (window.console[level] === 'undefined' || !window.console[level] || window.console[level] === null) {
-        window.console[level] = function () {};
+        window.console[level] = function () { };
```

Comment:
Why do we need this in the first place. Can't these be removed?

Reply:
I am not sure why this is present here.